src/utils.rs

```diff
@@ -40,19 +36,12 @@ pub fn print_error<S: AsRef<str>>(error: S, verbose: bool) {
 }
 
 /// Prints a warning to `stderr` in yellow.
-pub fn print_warning<S: AsRef<str>>(warning: S, verbose: bool) {
+pub fn print_warning<S: AsRef<str>>(warning: S, _: bool) {
```

Comment:
If we finally go for this option, sure, feel free to remove the `is_verbose` parameter.